.changes/tinymce/7.2.0.md

```diff
@@ -0,0 +1,27 @@
+## 7.2.0 - 2024-06-19
+
+### Added
+- Added `options.debug` API that logs the initial raw editor options to console. #TINY-10605
+- Added `referrerpolicy` as a valid attribute for an iframe element. #TINY-10374
+- New `onInit` and `stretched` properties to the `HtmlPanel` dialog component. #TINY-10900
+- Added support for querying the state of the `mceTogglePlainTextPaste` command. #TINY-10938
+- Added `for` option to dialog label components to improve accessibility. The value must be another component on the same dialog. #TINY-10971
+
+### Improved
+- Dialog slider components now emit an onChange event when using arrow keys. #TINY-10428
+- Accessibility for element path buttons, added tooltip to describe the button and removed incorrect `aria-level` attribute. #TINY-10891
+- Improve merging of inserted inline elements by removing nodes with redundant inheritable styles. #TINY-10869
+- Improved Find & Replace dialog accessibility by changing placeholders to labels. #TINY-10871
+
+### Changed
+- Replaced tiny branding logo with `Build with TinyMCE` text and logo. #TINY-11001
+
+### Fixed
+- Deleting in a `div` with preceeding `br` elements would sometimes throw errors. #TINY-10840
+- `autoresize_bottom_margin` was not reliably applied in some situations. #TINY-10793
+- CSS color values set to `transparent` were incorrectly converted to '#000000`. #TINY-10916
```

Comment:
As discussed in slack this morning this is a duplicate changelog that wasn't deleted from `main` after it was duplicated on `release/7`

- [ ] remove

Reply:
Done!